pkgs/applications/virtualization/colima/default.nix

```diff
@@ -0,0 +1,42 @@
+{
+  lib
+  , buildGoModule
+  , fetchFromGitHub
+  , installShellFiles
+  , go
+  , lima
+  , makeWrapper
+}:
+
+buildGoModule rec {
+  pname = "colima";
+  version = "0.2.2";
+
+  src = fetchFromGitHub {
+    owner = "abiosoft";
+    repo = "${pname}";
+    rev = "v${version}";
+    sha256 = "sha256-vWNkYsT2XF+oMOQ3pb1+/a207js8B+EmVanRQrYE/2A=";
+  };
+
+  nativeBuildInputs = [ go installShellFiles makeWrapper ];
```

Comment:
```suggestion
  nativeBuildInputs = [ installShellFiles makeWrapper ];
```

Reply:
`go` is not required because of `buildGoPackage`?